sormas-backend/src/main/java/de/symeda/sormas/backend/audit/LogSink.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.backend.audit;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.joran.JoranConfigurator;
+import ch.qos.logback.core.joran.spi.JoranException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.nio.file.InvalidPathException;
+
+/**
+ * Provides a configurable log sink for the SORMAS audit trail.
+ */
+public class LogSink {
+    private static final Logger logger = LoggerFactory.getLogger(LogSink.class);
+    private static LogSink instance;
+    private static final String CONFIG_PATH = "/opt/config/audit-logback.xml";
```

Comment:
Wouldn't it make sense to provide this value trough a `sormas.properties` property?
That way if the property is not configured we can deactivate the LogSink.
In case the LogSink is deactivated then `getAuditLogger` can probably provide a mock Logger which doesn't log anything.

Reply:
I was also thinking about this, maybe we do it in the future. On reason why I went with this for now is that we likely need to use POJOs, as the interceptors in the backend we are going to use are not running in an EJB/CDI context, so using anything which is a bean from there would likely be even more painful.

Reply:
Also pelase note this is not live for now. The important part is that I can programmatically read the config, which was the ultimate goal. Once I start with #8029, this will probably be absorbed anyway and then we can take care of the configuration aspect.

Reply:
I might have a solution....

Reply:
Fixed!